package.json

```diff
@@ -14,7 +15,8 @@
     "lint:stylelint": "stylelint \"src/**/*.{vue,htm,html,css,sss,less,scss}\"",
     "lint": "npm run lint:eslint && npm run lint:stylelint",
     "lint:fix": "npm run lint:eslint -- --fix && npm run lint:stylelint -- --fix",
-    "gendocs": "rm -rf srcdocs/ && jsdoc -c jsdoc.json"
+    "gendocs": "rm -rf srcdocs/ && jsdoc -c jsdoc.json",
+    "install": "if [ $(basename $(dirname $(pwd))) = \"node_modules\" ];then node build/buildModule.js;fi"
```

Comment:
Does this need to be called `install`? As it's just a different build option maybe `build:module` is better suited?

The build script filename has a redundant `build` in it, it could just be `build/module.js`. Then this script can just be `build/module.js`.

Reply:
"install" is an npm hook that triggers the build when the module is installed into the parent app. One of these is needed here:
https://docs.npmjs.com/misc/scripts

As per the second part of this request, I've renamed the build script accordingly.